### ISSUE TITLE
Image always uploads to froala

### ISSUE THREAD

**Issue by jackley** [froala/wysiwyg-editor#2466]
##### Expected behavior.
Image does not upload to froala

##### Actual behavior.
Image uploads to froala even with imageupload disabled

##### Steps to reproduce the problem.
@stefanneculai I just tried 2.7.4 with imageUpload false and it is still posting to i.froala and attempting to upload.

![image](https://user-images.githubusercontent.com/2819040/34598311-ee414ae6-f1b9-11e7-9ede-cb4cf404635e.png)

![image](https://user-images.githubusercontent.com/2819040/34598324-039beeaa-f1ba-11e7-8bdd-bdfcbacadcf9.png)

![image](https://user-images.githubusercontent.com/2819040/34598349-36a6174e-f1ba-11e7-8838-467e56d8c722.png)

##### OS.
Windows 10

##### Browser.
Chrome 63.0

**Comment by stefanneculai** [comment]
@jackley are you still seeing this if you turn off https://www.froala.com/wysiwyg-editor/docs/options#imageUploadRemoteUrls?

**Comment by aaclayton** [comment]
Hi @stefanneculai, I'm experiencing a very similar issue which I think is related. I'm observing that using the Image "by URL" option, instead of the image being embedded as-is from the target URL (my expected behavior) the image is sent to the `imageUploadURL` for upload. In my case I'm using a custom upload URL with special logic and I've observed a bunch of images get saved to my s3 bucket which were intended to simply be linked from external sources.

I tested including the diff from your previous post 724067d5e5b88c668f017f127bb0376dd5617ac1, but I'm still seeing the same behavior under the latest version of the file 367de39b0238616536988c1802af4e38d5735ff0.

**Expected Behavior**
When using `imageByURL`, the image simply embeds directly creating <img> tag with the `src` attribute equal to the provided URL.

**Observed Behavior**
POST request is sent to my `imageUploadURL` to save the image as if I had instead used `imageUpload`.

**OS**
Windows 10

**Browser** 
Version 63.0.3239.132 (Official Build) (64-bit)

I hope this can be prioritized because currently my S3 bucket is getting polluted with people's .gifs every time someone wants to post something from imgur :(
